catalystwan/integration_tests/feature_profile/sdwan/test_transport.py

```diff
@@ -121,11 +121,12 @@
 
 
 class TestTransportFeatureProfileModels(TestFeatureProfileModels):
-    @classmethod
-    def setUpClass(cls) -> None:
-        super().setUpClass()
-        cls.api = cls.session.api.sdwan_feature_profiles.transport
-        cls.profile_uuid = cls.api.create_profile("TestProfileService", "Description").id
+    def setUp(self) -> None:
```

Comment:
This is wrong. It will create and destroy session, feature profile for every testcase

Reply:
Yes. I want to remove all connected parcels to selected profile. Without it I cannot create new TransportVpnParcel.

Reply:
Can we make separete class TestTransportFeatureProfileTransportVpn?